src/Project.ts

```diff
@@ -207,21 +232,25 @@ export default class Project {
       triggers.map(({ trigger, target }) => {
         return trigger.start(target);
       })
-    );
+    ).then();
   }
 
-  get spritesAndClones() {
+  public get spritesAndClones(): Sprite[] {
     return Object.values(this.sprites)
-      .flatMap(sprite => sprite.andClones())
+      .flatMap((sprite) => sprite!.andClones())
       .sort((a, b) => a._layerOrder - b._layerOrder);
   }
 
-  get spritesAndStage() {
+  public get spritesAndStage(): [...Sprite[], Stage] {
```

Comment:
Is there an advantage to writing this this way instead of `(Sprite | Stage)[]`? What you have *feels* more descriptive, but if TypeScript isn't actually learning anything more from it (e.g. that there is always exactly one `Stage` in the array or that the `Stage` is always the last element) I think we should use the more generic description.

Reply:
I'll change the signature